doc/content/devices/best-practices/_index.md

```diff
@@ -0,0 +1,62 @@
+---
+title: "Best Practices"
+description: ""
+weight:
+---
+
+This section contains best practices for building connected devices which use {{% tts %}} as a Network Server.
+
+<!--more-->
+
+LoRaWAN devices should always comply to the [LoRaWAN specification](https://www.lora-alliance.org/lorawan-for-developers), with special regard to minimizing unnecessary join requests and duty cycle limitations.
+
+The goal of these best practices is to optimize individual devices (especially for battery consumption) and the network as a whole (to reliably and efficiently serve more devices with the same number of gateways).
+
+## Unnecessary Join Requests
+
+The LoRaWAN specification warns especially against systematic rejoin in case of network failure. A device should keep the result of an activation in permanent storage if the device is expected to be turned off during its lifetime.
+
+A device may temporarily lose connection with the network for many reasons: if the network server or gateways are suffering from an outage, if there’s no coverage in the area, etc. Whenever a device rejoins, it consumes the closest gateway’s airtime to emit the downlink - and if many devices in an area are rejoining at the same time (e.g. in case of a temporary gateway outage), this leads to network bloating in the area.
+
+## Limit Transmission Length, Payload Size, and Duty Cycle
+
+Limit the frequency of your transmissions to the minimum possible. Can you sample data once a day? Or even less?
```

Comment:
```suggestion
Limit the periodicity of transmissions to the minimum. Can you sample data once a day? Or even less?
```

frequency is confusing, because all packets are transmitted on particular frequency.